Sources/Build/describe().swift

```diff
@@ -194,9 +194,16 @@ extension Product {
 }
 
 private func extraImports() -> [String] {
-    //FIXME HACK
-    if let I = getenv("SWIFTPM_EXTRA_IMPORT") {
-        return ["-I", I]
+    // FIXME: This is a hack to allow -I include paths to be passed through to
+    //        swift-build and swift-test invocations. SWIFTPM_EXTRA_IMPORTS is
+    //        a semicolon-delineated list of absolute paths, for example:
+    //
+    //        SWIFTPM_EXTRA_IMPORTS="/path/to/foo;/path/to/bar" swift-build
+    //
+    //        This should be fixed such that it does not rely upon environment
+    //        variables.
+    if let includes = getenv("SWIFTPM_EXTRA_IMPORTS") {
```

Comment:
Is there any particular reason why this hack couldn't already be scrapped and replaced with additional `-Xswiftc` (or `-Xcc`? I'm not sure I understand which should be used when) parameters. Similar to https://github.com/apple/swift-package-manager/pull/172